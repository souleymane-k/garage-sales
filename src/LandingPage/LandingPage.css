*{
  box-sizing: border-box;
  color:#551A8B;
}


  .homePage {
    display: flex;
    flex-wrap: wrap;
    justify-content: space-between;
    background-color: cornsilk;
  }
  
  .item {
    width: 20%;
    /* width: 32%; */
    height:50px;
    padding-bottom: 18%; /* 32:18, i.e. 16:9 */
    /* margin-bottom: 2%; (100-32*3)/2 */
    /* margin: 2%; */
    
    
  }
  #box1{
    color:white;
    background-image: url(../images/table-1.jpg);
    background-size: 100%;
    background-repeat: no-repeat;
  }
  #box2{
    background-image: url(../images/chair-1.jpg);
    background-size: 100%;
    background-repeat: no-repeat;
  }
  #box3{
    background-image: url(../images/chair-2.jpg);
    background-size: 100%;
    background-repeat: no-repeat;
  }
  #box4{
    background-image: url(../images/fleur3.jpg);
    background-size: 100%;
    background-repeat: no-repeat;
  }
  #box5{
    background-image: url(../images/table-1.jpg);
    background-size: 100%;
    background-repeat: no-repeat;
  }
  #box6{
    background-image: url(../images/fleur2.jpg);
    background-size: 100%;
    background-repeat: no-repeat;
  }
  #box7{
    background-image: url(../images/table-1.jpg);
    background-size: 100%;
    background-repeat: no-repeat;
  }
  #box8{
    background-image: url(../images/Fleur.jpg);
    background-size: 100%;
    background-repeat: no-repeat;
  }
  #box9{
    background-image: url(../images/headSet.jpg);
    background-size: 100%;
    background-repeat: no-repeat;
  }
  /* .homePage{
    position:absolute;
    display: flex;
    flex-direction: row;
    text-align: center;
    max-width:100%;
    height: 100%;
    
  } */
  img {
    opacity: 0.5;
  }
  
  img:hover {
    opacity: 1.0;
  }

 h1{
    width:100%;
    color:white;
    margin: 20px;
 }  
  div h3{
   color:white;
 }

 .landingHeader{
  display: flex;
   /* width: 100%; */
   height:300px;
  /* background-image: url(../images/table-1.jpg);  */
  /* background-size: contain; */
  /* background-repeat: no-repeat; */
  margin-bottom: 5px;
  background-color: #FFF8DC;

 }

.firtssection{
    width:100%;
    border:solid;
    height:250px;
    padding:5px;
    /* border-radius: 5%; */
    margin:10px;
    background-color:#EAF6F6;
    color:black;
    /* background-image: url(../images/shoes-11.jpg);  */
    background-size: contain;
    background-repeat: no-repeat;
    
}

.secondsection{
  /* position:relative; */
    width:100%;
    border:solid;
    padding:5px;
    margin:10px;
    height:250px;
    /* border-radius: 5%; */
    background-color:#EAF6F6;
    color:black;
    /* background-image: url(../images/shoes-11.jpg);  */
    /* background-size: contain;
    background-repeat: no-repeat; */
}
.thirdsection{
  /* position:relative; */
    width:100%;
    border:solid;
    padding:5px;
    margin:10px;
    height:250px;
    /* border-radius: 5%; */
    background-color:#EAF6F6;
    color:black;
   
}
  /* ............@media Query...........................*/

 /* @media all and (min-width: 640px) {
       .homePage{
        display: flex;
        flex-direction: row;
        text-align: center;
        max-width:100%;
        margin-top: 40px;
        margin-left: 40px;
      }  */
    /* header {
      display: flex;
      height:100px;
      
     } */
  
     /* .logo {
      margin: 14px 0 14px 20px;
      padding-top: 0;
      font-size: 42px;
    }  */
    
     /* .navButton {
        display: flex;
        flex-direction: row;
        justify-content: center;
        padding-bottom: 5px;
        max-width: 100%;
        list-style-type: none;
        margin: 0px 0 0px 0px;
        text-align: right;
        background-color: red; */
      /* }  */
   /* }  */
  /* Media Query and type  for screen smaller or equal to 640px*/
  @media all and (max-width: 640px) {

    .homePage{
      display: flex;
      flex-direction: column;
      text-align: center;
      max-width:90%;
      margin: 2px 40px;
      /* margin-top: 150px; */
    }
    .firtssection {
      width: 90%;
      padding:5px;
      height:250px;
      flex-direction: row;
      margin-bottom:15px;
    }
    .secondsection {
        width: 90%;
        padding:5px;
        height:250px;
        flex-direction: row;
        /* margin-bottom:15px; */
      }
    h2{
        color:white;
        font-size: 20px;
        text-align: center;

    }
    p{
        justify-content: center;
        text-align:center;
    }
    h3{
        text-align: center;
        color:red;
    }
  
  }