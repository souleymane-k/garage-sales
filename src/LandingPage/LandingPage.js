import React from 'react';
import './LandingPage.css';
// import Souleymane from '../../souleymane-3.jpg'

export default function LandingPage() {
  // <img src={Souleymane} alt="maPhoto"></img>
  return (
    <section className='homePage'>
      {/* <img src={Souleymane} alt="maPhoto"></img> */}
      <div className="firtssection">
      <h3>What is Garage-Sales App?</h3>
      <p>Garage sales app is a tool for people who need to complete a quick sale of their house items such as sofa, tv, cell phones, and others online. It will allow to access more buyers and make good deals. </p>
      </div>
      <div className="secondsection">
      <h3>Why an App for Garage sales</h3>
      <p>This App responds to the need to make the neighboring sales more attractive and increase the target population. In addition, the garage sales app also responds to security needs where buyers and sellers can peacefully meet and complete their business. Please Register and then sign.</p>
      </div>
    </section>
  
    
  )
};