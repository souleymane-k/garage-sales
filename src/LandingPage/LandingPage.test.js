import React from 'react';
import ReactDOM from 'react-dom'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import LandingPage from './LandingPage';

describe(`LandingPage component`, () => {
  it('renders the complete form', () => {
    const wrapper = shallow(<LandingPage />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<LandingPage />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
})