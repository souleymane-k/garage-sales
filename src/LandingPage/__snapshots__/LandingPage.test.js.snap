// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage component renders the complete form 1`] = `
<div>
  <section
    className="landingHeader"
  >
    <div
      className="firtssection"
    >
      <h2>
        What is Garage-Sales App?
      </h2>
      <p>
        Garage sales app is a tool for people who need to complete a quick sale of their house items such as sofa, tv, cell phones, and others online. It will allow to access more buyers and make good deals. 
      </p>
    </div>
    <div
      className="secondsection"
    >
      <h2>
        Why an App for Garage sales
      </h2>
      <p>
        This App helps to make the neighboring sales more attractive and increase the target population. In addition, the garage sales app also responds to security needs where buyers and sellers can peacefully meet and complete their business. Please Register and then sign.
      </p>
    </div>
    <div
      className="thirdsection"
    >
      <h2>
        Why an App for Garage sales
      </h2>
      <p>
        Greate accessibility nationwide  to hundreds of items in a single platform. In addition, it is an exchange platform to create a tool of communication between salers and buyers.
      </p>
    </div>
  </section>
  <section
    className="homePage"
  >
    <div
      className="item"
      id="box1"
    >
      <h3>
        Living Room Table $200. Contact me at 555 666 6769
      </h3>
    </div>
    <div
      className="item"
      id="box2"
    >
      <h3>
        Dinning Chair $20 each
      </h3>
    </div>
    <div
      className="item"
      id="box3"
    >
      <h3>
        Chair with cuttion $15
      </h3>
    </div>
    <div
      className="item"
      id="box4"
    >
      <h3>
        Orange flower 
      </h3>
    </div>
    <div
      className="item"
      id="box5"
    >
      <h3>
        Why an App for Garage sales
      </h3>
    </div>
    <div
      className="item"
      id="box6"
    >
      <h3>
        Why an App for Garage sales
      </h3>
    </div>
    <div
      className="item"
      id="box7"
    >
      <h3>
        Why an App for Garage sales
      </h3>
    </div>
    <div
      className="item"
      id="box8"
    >
      <h3>
        Beautiful Flower
      </h3>
    </div>
    <div
      className="item"
      id="box9"
    >
      <h3>
        Pink Head Set for $10 
      </h3>
    </div>
  </section>
  <section
    className="lastSection"
  />
</div>
`;
